app/views/events/_applicants_overview.html.erb

```diff
@@ -53,11 +53,8 @@
 
   <div class="container-fluid no-padding">
     <div class="btn-group" role="group">
-      <%= link_to t('.accept_all'),
-          events_path, :class => 'btn btn-default'%>
-
-      <%= link_to t('.print_all'),
-                  print_applications_event_path(@event.id), :target => "_blank", :class => 'btn btn-default'%>
+      <%= link_to t('.accept_all'), event_accept_all_applicants_path(@event.id), :class => 'btn btn-default' %>
```

Comment:
As far as I know in the default case it doesn't make any different, because `to_param` will call `id`. But it is possible to override the `to_param` method to provide more readable pathes for example `www.example.com/events/1` could be changed to `www.example.com/events/mint_camp_2016`. Currently it is inconsistent in general so that changes like this will cause in a lot of changes either way.